src/pybamm/discretisations/discretisation.py

```diff
@@ -726,6 +726,8 @@ def process_dict(self, var_eqn_dict, ics=False):
         """
         new_var_eqn_dict = {}
         for eqn_key, eqn in var_eqn_dict.items():
+            if eqn_key == "Negative electrode open-circuit potential [V]":
+                print(eqn_key)
```

Comment:
needs to be removed

Reply:
that's weird, that shouldn't have come from this pr